internal/binutils/binutils_test.go

```diff
@@ -361,3 +362,31 @@ func TestLLVMSymbolizer(t *testing.T) {
 		}
 	}
 }
+
+func TestOpenMalformedELF(t *testing.T) {
+	// Test that opening a malformed ELF file will report an error containing
+	// the word "ELF".
+	bu := &Binutils{}
+	_, err := bu.Open(filepath.Join("testdata", "malformed_elf"), 0, 0, 0)
```

Comment:
Nit: merge with next line

Reply:
Can't because `err` is used later.

Reply:
Ah, sorry missed that. I'll merge once travis is happy (there were some failures, look transient, I restarted the builds).

Thanks again for the PR and bearing with my comments.

Reply:
No problem :-)